lms/djangoapps/instructor/views/registration_codes.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
I can't use the words TODO in the code, because it'll trigger a pylint violation (seriously). Filed: https://openedx.atlassian.net/browse/SOL-956